project.clj

```diff
@@ -2,9 +2,9 @@
 ;;;;
 ;;;; gorilla-repl is licenced to you under the MIT licence. See the file LICENCE.txt for full details.
 
-(defproject gorilla-repl "0.3.6"
+(defproject org.clojars.benfb/gorilla-repl "0.4.1"
```

Comment:
We should point these at the original dep on clojars, and below for the repo, before merging this PR.